migrations/2024-01-22-105746_lemmynsfw-changes/up.sql

```diff
@@ -1,2 +1,9 @@
-alter table local_site add column content_warning text;
-alter table community add column only_followers_can_vote boolean not null default false;
\ No newline at end of file
+ALTER TABLE local_site
+    ADD COLUMN content_warning text;
+
+ALTER TABLE local_site
+    ADD COLUMN auto_expand_images boolean NOT NULL DEFAULT FALSE;
```

Comment:
Seems okay, but I could see some potential conflicts as we add ViewMode to local_user_settings (Card, SmallCard, List, etc).  For example, the Card view would be the same as `auto_expand_images`

Edit: thinking about it more, maybe there should be a `default_site_viewmode` the same way there is a default_site_sort, that can be overridden by a local user's setting there. 

That would make `auto_expand_images` just a short-term hack that we would need to do away with.